### ISSUE TITLE
Suggestion of improvement to the Linux installer

### ISSUE THREAD

**Issue by tomaszdubiel18** [FirebirdSQL/firebird#7498]
Hello.
I would like to suggest you a change to the Linux installer. Windows Installer asks whether to enable Legacy authentication. Linux does not. It creates issues when a user not very well familiar with Firebird details enables Legacy_Auth and is not aware of the vulnerability related to the default password of SYSDBA user on Legacy_Auth, because even when he checks whether a new password works, he won't know that still old password is valid.
I would like to request a change so that a user when installing Firebird on Linux has a possibility to enable Legacy already at the installation so that changed password already works for both plugins.

**Comment by tomaszdubiel18** [comment]
If you propose to completely eliminate a SYSDBA user, that's also a good idea.

**Comment by aafemt** [comment]
Yes, that's exactly what I said: to remove predefined SYSDBA user from legacy auth tables.

**Comment by tomaszdubiel18** [comment]
Then it would be very worth to include in Release Notes to describe this change and give attention for Firebird admins to check whether there exists Legacy SYSDBA with masterkey password.

**Comment by tomaszdubiel18** [comment]
@aafemt, do you need to internally discuss whether to implement this change or not?

**Comment by tomaszdubiel18** [comment]
What @aafemt said is enough: to remove predefined SYSDBA user from legacy auth tables.
We install Firebird, we have only Srp SYSDBA - there is no need to create Legacy_Auth SYSDBA when enabling Legacy plugin.

**Comment by tomaszdubiel18** [comment]
@aafemt, can I ask you to discuss this topic internally among FB developers?

**Comment by aafemt** [comment]
I see no point in this: all opinions are already expressed.

**Comment by tomaszdubiel18** [comment]
But who can be the one that can be convinced and decide to do such change? :-) It can be done in FB 5 if it's a problem to do it in 3 or 4.

**Comment by aafemt** [comment]
Firebird is an open-source project. Anyone can submit a pull request.
